core/templates/dev/head/pages/exploration_player/PlayerTranscriptService.js

```diff
@@ -44,13 +48,15 @@ oppia.factory('playerTranscriptService', ['$log', function($log) {
   // happens if the current card offers feedback to the learner before they
   // carry on.
   var transcript = [];
+  var numOfAnswersSubmitted = 0;
```

Comment:
Nit: please use numAnswersSubmitted (for parity with other things like numHintsConsumed)